src/extension.ts

```diff
@@ -84,34 +84,36 @@ function openTemplateAndSaveNewFile(type: string, namespace: string, filename: s
 
     const templateFilePath = path.join(extension.extensionPath, 'templates', templatefileName);
 
-    vscode.workspace.openTextDocument(templateFilePath)
-        .then(doc => {
-            let text = doc.getText()
-                .replace('${namespace}', namespace)
-                .replace('${classname}', filename);
+    fs.readFile(templateFilePath, 'utf8', (errTryingToCreate, fileContent) => {
```

Comment:
errTryingToCreate - Error name is wrong here now, should be something like 'errorTryingToReadFile' or 'errorReading'